code/src/Components/OfficeQuestion.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+export const OfficeQuestion = (props) => {
+    const {step, officeInput, onOfficeInputChange, onStepChange, onStepBackChange} = props
+    const progress = (step/8)*100
+    return(
+        <div className="container">
+            <p className="labelStyle">Which one of our officeswould you like to work?</p>
+                <div className="offices">
+                    <div>
+                        <label className="radioBtnContainer">Helsinki
+                            <input 
+                            name={officeInput}
+                            type="radio" 
+                            value='Helsinki'
+                            onChange={onOfficeInputChange}/>
+                            <span className="checkmark"></span>
+                        </label>
+                    </div>
+                    <div>
+                        <label className="radioBtnContainer">Barcelona
+                            <input 
+                            name={officeInput}
+                            type="radio" 
+                            value='Barcelona'
+                            onChange={onOfficeInputChange}/>
+                            <span className="checkmark"></span>
+                        </label>
+                    </div>      
+                    <div>
+                        <label className="radioBtnContainer">Remotely
+                            <input 
+                            name={officeInput}
+                            type="radio" 
+                            value='remotely'
+                            onChange={onOfficeInputChange}/>
+                            <span className="checkmark"></span>
+                        </label>
+                    </div>  
+                </div>
+
+                <div className="buttons">
+                <button className="button"
+                    disabled={officeInput===''}
+                    onClick={onStepChange}
+                    >Next <span className="arrow right"></span></button>
+                <button className="button"
+                    onClick={onStepBackChange}
+                    ><span className="arrow left"></span> Back</button>
+                
+                </div>
+                <div className="progress-container">
+                <div style={{width: `${progress}%`}} className="progress-value">
+                </div>
+            </div>
+        </div>
```

Comment:
There is nothing wrong with this approach but if you want to you could watch the video from tech to learn with Jennie, she talks about creating an array with the different options and then mapping over them so that you don't need to make separate labels/inputs for each option and just use one for all of them! could be something to look up and try for the next time. :)